org.mythtv.MythTV.metainfo.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.mythtv.MythTV</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0-or-later</project_license>
+  <name>MythTV</name>
+  <summary>
+    MythTV is a Free Open Source software digital video recorder (DVR) project distributed under the terms of the GNU GPL
+  </summary>
```

Comment:
reduce the summary, drop the license and name no need to repeat. also summary should not have break `<summary>foobar</summary>`